### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mikemcd3912**
Expanding test job requirements to cover repeatable testing using CronJobs

*Issue #, if available:*

*Description of changes:*
1. Updated Acceptable job types in Requirement 5 to include CronJobs
2. Updated repeatability requirements in Requirement 6. to include CronJob run frequency 
3. Added CronJob Example link 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,12 +42,12 @@ metadata:
 2. Functional test should validate the functionality of the ISV product and describe what the ISV product does
 3. Healthchecks, service endpoints checks or any other technical checks do not represent sufficient coverage required for the functional test
 4. Functional test should be wrapped as a container, container image should be published on ECR, and/or provide evidence of successful recent vulnerability scan
-5. Functional test must be implemented as a Kubernetes Job and any non-zero exit status of the job execution will be considered a failure
-6. Functional test must be repeatable. That means that if the job has executed before successfully and no changes were applied, we expect to run it continuously and mark the product as failure if the test job starts producing failures even if previous executions against the same environment were successful
+5. Functional test must be implemented as a Kubernetes Job or CronJob and any non-zero exit status of the job execution will be considered a failure
```
Comment (elamaran11) [inline README.md:45]: remove `Job or`

File: README.md [hunk 2]
```diff
@@ -42,12 +42,12 @@ metadata:
...
 7. Functional test should not require elevated security permissions, such as cluster roles, privileged mode, non-ephemeral storage
 8. Functional test should be submitted under `eks-anywhere-common/testers` (runs on all platforms) or under your respective environment folder such as `eks-anywhere-snow/testers` (e.g. `eks-anywhere-snow/testers/<orgname>/<productname>`)
 
-Refer the example [here](https://github.com/aws-samples/eks-anywhere-addons/tree/main/eks-anywhere-common/Testers/Hashicorp/Vault/kvJob.yaml) for functional test job.
+Refer the example [here](https://github.com/aws-samples/eks-anywhere-addons/tree/main/eks-anywhere-common/Testers/Hashicorp/Vault/kvJob.yaml) for functional test job, or the example [here](https://github.com/aws-samples/eks-anywhere-addons/blob/main/eks-anywhere-common/Testers/Komodor/komodor-testjob.yaml) for functional test cronjob.
```
Comment (elamaran11) [inline README.md:50]: Job example can be removed. Just have CJ example.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@mikemcd3912 Have a Minor comment.

File: README.md (hunk 2 above)
Comment (mikemcd3912) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (mikemcd3912) [reply README.md]: Updated

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**